Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -99,6 +99,19 @@ - (NSInteger)indexOfImage:(uint64_t)address
     return -1; // Address not found
 }
 
+- (nullable NSString *)pathForImage:(NSString *)binaryName
```

Comment:
`m`: Although most of the code here is covered by tests in `SentryUIViewControllerSwizzlingTests`, I think we should add some tests to `SentryBinaryImageCacheTests` to validate that we use `hasPrefix` and that this method returns `nil` if we don't find anything.

Reply:
I can't find any tests for what I specified above ⬆️. @brustolin, am I missing something, or did you resolve this comment by mistake?

Reply:
https://github.com/getsentry/sentry-cocoa/blob/cff8050429f3844e332dbf93aafd85769f2fbebc/Tests/SentryTests/SentryBinaryImageCacheTests.swift#L100-L109